backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -263,7 +263,7 @@ export class AdminResolver {
 
     const userIds = pendingCreations.map((p) => p.userId)
     const userCreations = await getUserCreations(userIds)
-    const users = await User.find({ id: In(userIds) })
+    const users = await User.find({ where: { id: In(userIds) }, withDeleted: true })
```

Comment:
When we not collect the deleted users as well, the UI shows empty name, email, etc.